Source/Server/Structs/StartStruct.h

```diff
@@ -1,7 +1,7 @@
 #ifndef SERVERSTARTOPTIONS_H
 #define SERVERSTARTOPTIONS_H
 
-struct server_start_options
+struct server_args
```

Comment:
1 tiny thing here. Typedef it. So in Main.c it can be initialized such as server_args args = {.....}